mne/viz/utils.py

```diff
@@ -543,17 +552,60 @@ def _plot_raw_time(value, params):
         params['hsel_patch'].set_x(value)
 
 
+def _change_channel_group(step, params):
+    """Deal with change of channel group."""
+    from .raw import _radio_clicked
```

Comment:
if this created a circular import, `_radio_clicked` should probably be moved here instead if possible